### DISCUSSION TITLE
Install php from brew

### DISCUSSION THREAD

**Discussion by chapa38** [Homebrew/discussions#1910] [Getting started]
I install brew and php 7.4 in MacOS Big Sur ver 11.4 When I check the version php -v. Still missing. Any comment please.
Message: chapa@frommm ~ % php -v
WARNING: PHP is not recommended
PHP is included in macOS for compatibility with legacy software.
Future versions of macOS will not include PHP.
PHP 7.3.24-(to be removed in future macOS) (cli) (built: May  8 2021 09:40:37) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.3.24, Copyright (c) 1998-2018 Zend Technologies

**Comment by SMillerDev** [comment]
`brew install php` would have told you that it's not in the PATH by default. After you add it to the PATH it'll show php 7.4 there.

**Comment by NeoVG** [comment]
You can also use `brew link php@7.4` to create symbolic links to PHP binaries in `/usr/local/bin` and `/usr/local/sbin`.
